### ISSUE TITLE
Test C888: "Authentication required" preventing from entering topic description

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#364]
On mobile, android, chrome, when creating a discussion topic, when trying to enter the description as soon as I clicked on the field it said "Authentication required" as per screenshot below, and there was no way to type anything. 

I was able to click next and continue though, which I would have thought shouldn't be possible as it's a field with a red asterisk. So I wonder if the mandatory fields are working as they should? Or it considered the "Authentication required" message to be content there?  

![Screenshot_20231117-094317_Chrome](https://github.com/citizenos/citizenos-fe/assets/61320526/1d707bb0-2510-400f-b1e2-e4bf5917baa3)

**Comment by BeccaMelhuish** [comment]
Got this on laptop, Windows, Chrome too, in case this is helpful :) 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/ef2cb0bb-4bc7-4d75-9d2b-0b0c4f416e70)

**Comment by BeccaMelhuish** [comment]
@anettlinno @ilmartyrk for me this is a high priority one to get fixed before testing, as it quite interrupts/blocks the flow of usage, added the label accordingly :)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish @anettlinno not sure why this is happening, it seems to be quite random. I've managed to get this message only once and closing/re-opening the description tab fixed it.

**Comment by BeccaMelhuish** [comment]
As mentioned earlier I haven't had this problem for a while either, tried on mobile this afternoon too and was fine :)

**Comment by anettlinno** [comment]
Triage 95. As could not replicate, then closing the issue with no further actions.

**Comment by BeccaMelhuish** [comment]
Re-opening as it came up in a user test:

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/5c54f030-f795-4b6c-8548-c8ea878b0e46)

**Comment by BeccaMelhuish** [comment]
Just for info it came up in the user test yesterday as well by the way! 

But when we got to the Preview step of the create process and then went back to the Compose tab, she was then able to add it in (i.e. it all looked and worked as it should).

**Comment by BeccaMelhuish** [comment]
And in user test today, quite a common one it seems :)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish I think it has something to do with Etherpad database getting full.  Allthough it has 10000 row limit it still allows sometimes to add new rows so it got over the limit. I think it actually got to the limit yesterday as there were almost11000 rows when I got to cleaning up. So I am closing this issue again

**Comment by BeccaMelhuish** [comment]
@ilmartyrk It happened again in a test today (mobile, android, Chrome) and I know you just cleared the database up a bit yesterday - so am opening again in case you think it wasn't that. Happened at 14.25 your time in case that helps see how full it was/wasn't :)  

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/a25eed5e-0a5e-4145-a394-fe4247a6663f)

**Comment by BeccaMelhuish** [comment]
@ilmartyrk Moved to in progress in case the above changes anything :)

**Comment by ssin1901** [comment]
I could not replicate the issue. Let's create a new issue if it happens again.

**Comment by BeccaMelhuish** [comment]
@ilmartyrk it happened again in the test today (around 14.40 your time), so just reopening to be sure it's the problem you thought it might be above :) 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/9d51b3d0-585b-48db-ade0-b042543718e1)

**Comment by ilmartyrk** [comment]
It is quite random, but there is a new version 2 of Etherpad out, I hope it will solve the problem when I have time to upgrade to it.

**Comment by BeccaMelhuish** [comment]
Oh that's good then, fingers crossed! It's odd that it's always this particular field (never title or intro)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish that is because for that description field we use Etherpad in iframe, it is fully separate "app" that we use and we have an authorization process between API and Etherpad, but for some reason it sometimes doesn't work properly and then this issue happens

**Comment by BeccaMelhuish** [comment]
Happened again in today's test just FYI. Seems it often happens the first time, when creating a new topic, but that when you later go back to edit, it works. (So they are able to add the text later).

**Comment by BeccaMelhuish** [comment]
@ilmartyrk Can confirm I'm seeing a very strong pattern (in test) - almost every single new topic has this bug when they first create the topic, so the first time they get to this field. But whenever they go back to this field later, e.g. editing or in draft, it's OK. 

It happened the same to me too the other day (can't remember if in live or test sorry), I saved it as draft, opened it up again and then the bug was gone :)

**Comment by BeccaMelhuish** [comment]
@ilmartyrk should I keep updating you when it's happening in tests (it still is currently, and still a pattern that it's always the first time they go to it, but fine after that), or no need?

**Comment by BeccaMelhuish** [comment]
Additional info from today's test: a user entered title, intro etc. and then got the bug. Saved topic as draft and then came back to it. The bug was still there. So seems the pattern may be that the bug is there until the user has moved on to a later step in the create flow, but when they haven't got past step 1, it is there. (I would have to double check the videos of all the other occurrences to be sure if this pattern is correct, but that's my hunch from memory).

**Comment by BeccaMelhuish** [comment]
OK scrap that suggestion of a pattern actually, it just happened to me when editing a topic I made a week ago, so very much not new.. 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/56ef008f-7d91-47e3-9f4f-d1783a24802a)

**Comment by ilmartyrk** [comment]
Updated Etherpad version in live too. hopefully this issue won't happen again, so closing this issue and lets reopen if needed

**Comment by BeccaMelhuish** [comment]
@ssin1901 should this be in testing column? Move it back if not :)

**Comment by ssin1901** [comment]
@ilmartyrk @BeccaMelhuish haven't been able to replicate this issue since the Etherpad update. Can we close the issue and move to done?

**Comment by BeccaMelhuish** [comment]
@ssin1901 @ilmartyrk I'm sorry to report that it just happened again, so will re-open:

![image](https://github.com/user-attachments/assets/3cff22be-e7c8-4635-8e48-cc23cfe8a607)

I'm on desktop, Chrome (and with sometimes dodgy internet connection, if that's relevant)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish has this happened again or can we close it

**Comment by BeccaMelhuish** [comment]
I don't think so for me, but I'm not sure if I've made any topics with topic text in them since then.. Maybe @ssin1901 can test?

**Comment by ssin1901** [comment]
@ilmartyrk I had opened an issue about this -  https://github.com/citizenos/citizenos-fe/issues/1570 last week. Haven't seen it other topics though.

**Comment by BeccaMelhuish** [comment]
Just noting that this happened again for @ssin1901 [here](https://github.com/citizenos/citizenos-fe/issues/1612) on Sep 27, but now closed that issue as a duplicate :)

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @ssin1901 I'm unsure which column to keep this in, as it's a high priority one in terms of being important to solve, but I understand it's also a 'mystery' issue, in that we don't know how to fix it? 

For now I will assign @ilmartyrk and put it 'in prep' to be looked into further?

**Comment by BeccaMelhuish** [comment]
Just a note that this happened again for me just now (in live, Chrome, Windows, laptop): 

![Image](https://github.com/user-attachments/assets/2092e859-c1f3-4c4e-9dba-0f6f7f46e6ed)

**Comment by BeccaMelhuish** [comment]
Ilmar: needs research, fix not clear. May be to do with syncing, add it to the syncing group issues.

**Comment by BeccaMelhuish** [comment]
Adding 'grouped' label and moving to 'no status' column as it is now covered in #1788

**Comment by ssin1901** [comment]
Issue replicated in Test environment, as per comment in #1396

**Comment by BeccaMelhuish** [comment]
Happened in a test in live recently too

**Comment by BeccaMelhuish** [comment]
Recent user testing shows this is still a serious bug for users in live
